dev/builder/build-config.js

```diff
@@ -50,8 +50,8 @@ var CKBUILDER_CONFIG = {
 		'tests'
 	],
 	plugins : {
-		'a11yhelp' : 1,
-		'autogrow' : 1,
+		'a11yhelp' : 0,
+		'autogrow' : 0,
```

Comment:
Policystat doesn't use these, so they can be excluded from ckeditor.js (the main bundle)